indra/llcharacter/llik.cpp

```diff
@@ -147,27 +148,33 @@ F32 compute_clamped_angle(F32 angle, F32 min_angle, F32 max_angle)
 void LLIK::Joint::Config::setLocalPos(const LLVector3& pos)
 {
     mLocalPos = pos;
-    mFlags |= FLAG_LOCAL_POS;
+    mFlags |= CONFIG_FLAG_LOCAL_POS;
 }
 
 void LLIK::Joint::Config::setLocalRot(const LLQuaternion& rot)
 {
     mLocalRot = rot;
     mLocalRot.normalize();
-    mFlags |= FLAG_LOCAL_ROT;
+    mFlags |= CONFIG_FLAG_LOCAL_ROT;
+}
+
+void LLIK::Joint::Config::setLocalScale(const LLVector3& scale)
+{
+    mLocalScale = scale;
+    mFlags |= CONFIG_FLAG_LOCAL_SCALE;
 }
```

Comment:
We always intended to add the ability to adjust the scale of a joint via Puppetry.  I'm adding it now, mostly because I had to fix some scale-related math for avatar models that use a non-unitary scale at the **LLJoint** context, and once I started digging into that I figured out what it would take to support Puppetry scale.